Please bundle pdf.js

Could you please bundle pdf.js? We are behind a firewall and cloning from github fails:

```
Installing “pdf-view@0.10.0” failed.Hide output…
...
npm ERR! git clone git://github.com/mozilla/pdf.js.git github.com[0: 192.30.252.129]: errno=Operation timed out
npm ERR! Error: Command failed: Cloning into bare repository '/Users/laktak/.atom/.node-gyp/.atom/.apm/_git-remotes/git-github-com-mozilla-pdf-js-git-09e1fcd2'...
npm ERR! fatal: unable to connect to github.com:
```

--- a/package.json
+++ b/package.json
@@ -14,6 +14,6 @@
   "dependencies": {
     "fs-plus": "2.x",
     "pathwatcher": "^2.0",
-    "pdf.js": "git://github.com/mozilla/pdf.js.git#4834f1c28924b3322f00b8383ddb8478193be930"
+    "pdf.js": "git+https://github.com/mozilla/pdf.js.git#4834f1c28924b3322f00b8383ddb8478193be930"
   }
 }

